### PR TITLE
Users/ajya/handling error cmd

### PR DESCRIPTION
**Pull request by Ajay-MS**

### CODE REVIEW

File: Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts [hunk 1]
```diff
@@ -243,4 +243,20 @@ function canUseWebDeploy(useWebDeploy: boolean) {
...
+ * 1. Checks if msdeploy during execution redirected any error to 
+ * error stream ( saved in error.txt) , display error to console
+ * 2. Checks if there is file in use error , suggest to try app offline.
+ */
+function redirectMSDeployErrorToConsole() {
+    var msDeployErrorFilePath = tl.getVariable('System.DefaultWorkingDirectory') + '\\error.txt';
+    if(tl.exist(msDeployErrorFilePath)) {
+        var errorFileContent = fs.readFileSync(msDeployErrorFilePath);
+        if(errorFileContent.toString().indexOf("ERROR_INSUFFICIENT_ACCESS_TO_SITE_FOLDER") !== -1){
+            tl.warning(tl.loc("Trytodeploywebappagainwithappofflineoptionselected"));
```
Comment (mvvsubbu) [inline Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts]: Why this has to be warning. If this is warning my task will succeed, is this the outcome we wanted

Comment (Ajay-MS) [inline Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts]: Actual reason of error is being shown as error, "try app offline" is a suggestion that's why being shown as warning.

Comment (kmkumaran) [inline Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts]: Do you want to append the guidance message with errorFileContent and printing them together in tl.error below?

File: Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts [hunk 2]
```diff
@@ -243,4 +243,20 @@ function canUseWebDeploy(useWebDeploy: boolean) {
     return (useWebDeploy || win);
 }
 
+/**
+ * 1. Checks if msdeploy during execution redirected any error to 
+ * error stream ( saved in error.txt) , display error to console
+ * 2. Checks if there is file in use error , suggest to try app offline.
+ */
+function redirectMSDeployErrorToConsole() {
+    var msDeployErrorFilePath = tl.getVariable('System.DefaultWorkingDirectory') + '\\error.txt';
```
Comment (kmkumaran) [inline Tasks/AzureRmWebAppDeployment/azurermwebappdeployment.ts]: should this error.txt file be cleaned up in the end?

**Pull request merged**